include/stellar/io/import_sequence.hpp

```diff
@@ -0,0 +1,163 @@
+#pragma once
+
+#include <seqan/seq_io.h>
+
+namespace stellar
+{
+
+using namespace seqan2;
+
+template <typename TId>
+inline bool
+_checkUniqueId(std::set<TId> & uniqueIds, TId const & id)
+{
+    TId shortId;
+    typedef typename Iterator<TId const>::Type TIterator;
+
+    TIterator it = begin(id);
+    TIterator itEnd = end(id);
+
+    // (cut at first whitespace)
+    while (it != itEnd && *it > ' ')
+    {
+        appendValue(shortId, *it);
+        ++it;
+    }
+
+    if (uniqueIds.count(shortId) == 0)
+    {
+        uniqueIds.insert(shortId);
+        return 1;
+    }
+
+    return 0;
```

Comment:
Or even shorter:
```suggestion
    auto [it, added] = uniqueIds.insert(shortId);
    return added;
```

Maybe a little extreme:
```suggestion
    return std::get<bool>(uniqueIds.insert(shortId));
```